SignalServiceKit/src/Messages/OWSMessageManager.m

```diff
@@ -341,17 +364,32 @@ - (void)handleIncomingEnvelope:(OWSSignalServiceProtosEnvelope *)envelope
 
 - (void)handleIncomingEnvelope:(OWSSignalServiceProtosEnvelope *)envelope
             withReceiptMessage:(OWSSignalServiceProtosReceiptMessage *)receiptMessage
+                   transaction:(YapDatabaseReadWriteTransaction *)transaction
 {
     OWSAssert(envelope);
     OWSAssert(receiptMessage);
+    OWSAssert(transaction);
+
+    PBArray *messageTimestamps = receiptMessage.timestamp;
```

Comment:
naming nit: can we make this `receiptMessage.timestamps` (plural) since it's an array and we use timestamp (singular) all over the place.

Reply:
I think this is a bug in the obj-c code generation.  I think the protobuf style is that "repeated" properties should have non-plural names and the generated code should pluralize them as appropriate.

* addTimestamp:<single value>.
* addTimestamps:<single value>.
* getTimestampAtIndex:
* getTimestamps

`protobuf-objc` appears to get the pluralization right for the setters in the builders but not this kind of getter.

Reply:
I think we just have a bug in our proto file.

Reply:
I got this!

Reply:
oh. Sorry, I see what you're saying... we might as well leave this as is since we're already inconsistent about naming repeated fields plural (e.g. members, attachments, read, iceUpdate)